test/components/FeatureFlagRenderer.test.tsx

```diff
@@ -286,8 +304,10 @@ describe("components/FeatureFlagRenderer", () => {
     it("keeps track of mount state on instance", () => {
       const wrapper = mountRender();
       const wrapperInstance = wrapper.instance();
+      // @ts-ignore
```

Comment:
wasn't sure how to fix this without ignoring it, any ideas are welcomed

Reply:
@AlinaDogariu can use `expect((wrapperInstance as FeatureFlagRenderer)._isMounted).toBe(true);`